CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -13,6 +13,7 @@ private typealias Section = Styleguide.Section
 public struct MyCoinView: View {
 
     @EnvironmentObject var myCoinState: MyCoinState
+    @State private var selectedTimeFrameItem: TimeFrameItem = .init(timeFrame: .oneDay)
```

Comment:
Optional, maybe adding a line break above for grouping different type of var, just a code presentation improvement 😁

Reply:
🙏 I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/99/commits/1790e7a7915af1aece3b9751017a8ee0f4628d48